Framework/PythonInterface/test/python/plugins/algorithms/MuonMaxEntTest.py

```diff
@@ -1,32 +1,57 @@
 from __future__ import (absolute_import, print_function)
 
 import numpy as np
-
+import math
 import unittest
 from mantid.simpleapi import *
 from mantid.api import *
 
+
 class MuonMaxEntTest(unittest.TestCase):
+
     def genData(self):
-         x_data = np.linspace(0,30.,100)
-         e_data = np.cos(0.0*x_data)
-         y_data = np.sin(2.3*x_data+0.1)*np.exp(-x_data/2.19703)
-         inputData= CreateWorkspace(DataX=x_data,DataY=y_data,DataE=e_data,UnitX='Time')
-         return inputData
+        x_data = np.linspace(0, 30., 100)
+        e_data = np.cos(0.0 * x_data)
+        y_data = np.sin(2.3 * x_data + 0.1) * np.exp(-x_data / 2.19703)
+        inputData = CreateWorkspace(
+            DataX=x_data,
+            DataY=y_data,
+            DataE=e_data,
+            UnitX='Time')
+        return inputData
 
     def genData2(self):
-         x_data = np.linspace(0,30.,100)
-         e_data = []
-         y_data = []
-         for xx in x_data:
-               y_data.append(np.sin(2.3*xx+0.1)*np.exp(-xx/2.19703))
-               e_data.append(np.cos(0.2*xx))
-         for xx in x_data:
-               y_data.append(np.sin(4.3*xx+0.2)*np.exp(-xx/2.19703))
-               e_data.append( np.cos(0.2*xx))
+        x_data = np.linspace(0, 30., 100)
+        e_data = []
+        y_data = []
+        for xx in x_data:
+            y_data.append(np.sin(2.3 * xx + 0.1) * np.exp(-xx / 2.19703))
+            e_data.append(np.cos(0.2 * xx))
+        for xx in x_data:
+            y_data.append(np.sin(4.3 * xx + 0.2) * np.exp(-xx / 2.19703))
+            e_data.append(np.cos(0.2 * xx))
 
-         inputData= CreateWorkspace(DataX=x_data,DataY=y_data,DataE=e_data,NSpec=2,UnitX='Time')
-         return inputData
+        inputData = CreateWorkspace(
+            DataX=x_data,
+            DataY=y_data,
+            DataE=e_data,
+            NSpec=2,
+            UnitX='Time')
+        return inputData
+
+    def genData3(self):
```

Comment:
Again, is there maybe a more descriptive name for this than `genData3`, such as `genDataWithDeadDetectors`?